File: src/main/java/com/adamkali/simpleide/editor/lang/tokens/keyword/KeywordToken.java
package com.adamkali.simpleide.editor.lang.tokens.keyword;

import com.adamkali.simpleide.editor.lang.tokens.Token;
import com.adamkali.simpleide.preferences.EditorColors;

public class KeywordToken extends Token {
    public KeywordToken(String text) {
        super(EditorColors.KEYWORD_BG_COLOR, EditorColors.KEYWORD_FG_COLOR, text);
    }
}

File: src/main/java/com/adamkali/simpleide/project/lang/tokens/keyword/KeywordToken.kt
package com.adamkali.simpleide.project.lang.tokens.keyword

import com.adamkali.simpleide.preferences.EditorColors
import com.adamkali.simpleide.project.lang.tokens.Token

abstract class KeywordToken(text: String) : Token(text, EditorColors.KEYWORD_BG_COLOR, EditorColors.KEYWORD_FG_COLOR) {
}